aws-js-webserver/README.md

```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# AWS EC2 Web Server
+# Python Web Server Using Amazon EC2
```

Comment:
The previous example is `AWS Web Server Component` while this one is `Python Web Server Using Amazon EC2`. They are almost the same thing, so I would structure the title in a similar way.
I would do something like `Web Server (Component) Using Amazon EC2`